portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,81 @@
+        <br><br>
+        For a couple of years, I've loved to be part of student clubs related to my career, teaching programming fundamentals to middle schoolers and participating at the hacking community of MLH as a hackathon's organizer    
+      </p> 
+      
+      <h3 style="text-align:center;">Social Profiles </h3>
+      <p style="text-align:center;">
+      <a href="https://www.instagram.com/marifer_salas/?hl=es" >
+        <img src="/images/instagram.png" alt="Instagram" style="width:100px; height:100px; padding:10px"/>
+      </a>
+      <a href="https://www.linkedin.com/in/mariafernanda-salas-martinez/">
+        <img src="/images/linkedin.png" alt="LinkedIn" style="width:100px; height:100px; padding:10px"/>
+      </a>
+      <a href="https://github.com/MariferSalas">
+        <img src="/images/github.png" alt="Github" style="width:100px; height:100px; padding:10px"/>
+      </a>
+      </p>
+      
+      <h3 style="text-align:center;">Random Quotes </h3>
+      <p>Click here to get a random quote from my favorite TV Show (Grey's Anatomy):</p>
+      <button onclick="addRandomGreeting()">Get Quote</button>
       <div id="greeting-container"></div>
+      
+      <h2>Projects & Experiences</h2>
+      <h3>ACM Monterrey</h3>
+        <p>
+        ACM MTY stands for Association for Computing Machinery (ACM) Student Chapter in Monterrey.
+        </p>
+        <ul>
+            <li>Logistics Coordinator, 2019</li>
+            <li>President, 2020</li>
+            <li>Social and Student Responsibility Coordinator, 2021</li>
+        </ul>
+
+      <h3>HackMTY</h3>
+        <ul>
+            <li>Logistics Coordinator, 2019</li>
+                <dd>Hosted Mexico's biggest student hackathon attended by 500 students.</dd>
+            <li>President, 2020</li>
+                <dd>Hosted HackMTY’s first digital hackathon, attended by 400 students from all over Mexico.</dd>
+            <li>Logistics Coordinator, 2021</li>
+                <dd>Soon...</dd>
+        </ul>
+      <h3>Programando un Mexico Mejor</h3>
+        <p>Coordinator of PUMM, a web development course for middle school students, 2020 & 2021.</p>
+      <h3>ApportaTeen</h3>
+        <p>Coordinator of AportaTeen: Competitive Programming Course, a Java programming basics course for middle school and junior high school students, 2019 & 2020.</p>
+      <h3>Women In Tech</h3>
+        <p>Instructor of WIT Code,  a course for young girls aged 12 to 15 interested in computer science, 2019 & 2020.</p>
+      
+      <div class="image-thumbnails">
+        <img src="images/ACMMTY2019.jpg"/>
+        <img src="images/AXT2019.jpeg"/>
+        <img src="images/ACMMTY_2020.png"/>
+        <img src="images/ACMMTY2021.jpg"/>
+        <img src="images/HM2019.PNG"/>
+        <img src="images/HM2020.jpeg"/>
+        <img src="images/PUMM.png"/>
+        <img src="images/AXT2020.jpeg"/>
```

Comment:
It might be nice to add the "alt" tags here as well for each image, just in case users have slow connections or other issues with images loading. This is also important for accessibility because visually impaired users won't know what the image is showing.

Reply:
You are totally right, thanks for reminding me of it :)